src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,50 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortSubmission
```

Comment:
This doesn't make sense as a logical unit.  I don't think the abort logic needs to be wrapped in its own React class, but if you want to do it that way, then it makes more sense to make an AbortButton class and not also encompass the status display.